saleor/graphql/order/types.py

```diff
@@ -255,6 +255,9 @@ class OrderLine(CountableDjangoObjectType):
     translated_variant_name = graphene.String(
         required=True, description="Variant name in the customer's language"
     )
+    allocations = graphene.List(
+        Allocation, description="List of allocations across warehouses.",
```

Comment:
```suggestion
        graphene.NonNull(Allocation), description="List of allocations across warehouses.",
```
We shouldn't allot to return the list with nulls.